src/app/containers/RadioSchedule/index.jsx

```diff
@@ -22,6 +27,10 @@ const RadioScheduleContainer = ({ endpoint }) => {
     fetchRadioScheduleData(endpoint);
```

Comment:
Should we consider putting something in place to avoid the call to fetch schedule data? I assume it's still firing here, but won't be needed if the feature is disabled.

Reply:
Yes, we should. Added https://github.com/bbc/simorgh/pull/5275/files#diff-8c93bbb92b2b3fbf9b7861d8c8836f8eR27-R29 to only call he fetch function when the toggle is enabled